lib/cocoapods/resolver.rb

```diff
@@ -395,6 +400,17 @@ def handle_resolver_error(error)
       raise Informative, error.message
     end
 
+    def plat?(dg, req, spec)
```

Comment:
Can we use some more explanatory variable names to make this readable to people who didn't write the resolver/this particular piece of code? Ex: `dependency_graph` not `dg`